params/ekf_template.yaml

```diff
@@ -2,7 +2,7 @@
 # computation until it receives at least one message from one of the inputs. It will then run continuously at the
 # frequency specified here, regardless of whether it receives more measurements. Defaults to 30 if unspecified.
 frequency: 30
-
+silentchecktf: false
```

Comment:
Can we perhaps rename this parameter? Something like `silent_tf_failure` would be more in line with the parameter scheme, I think.